app/lib/features/chat_ui_showcase/models/convo_showcase_data.dart

```diff
@@ -0,0 +1,315 @@
+import 'package:acter/features/chat_ui_showcase/models/mocks/mock_convo.dart';
+import 'package:acter/features/chat_ui_showcase/models/mocks/mock_room.dart';
+import 'package:acter/features/chat_ui_showcase/models/mocks/mock_user.dart';
+
+class MockChatItem {
+  final String roomId;
+  final MockRoom mockRoom;
+  final MockConvo mockConvo;
+  final List<MockUser>? typingUsers;
+
+  MockChatItem({
+    required this.roomId,
+    required this.mockRoom,
+    required this.mockConvo,
+    required this.typingUsers,
+  });
+}
+
+MockChatItem createMockChatItem({
+  required String roomId,
+  required String displayName,
+  String? notificationMode,
+  List<String>? activeMembersIds,
+  bool? isDm,
+  bool? isBookmarked,
+  int? unreadNotificationCount,
+  int? unreadMentions,
+  int? unreadMessages,
+  List<MockUser>? typingUsers,
+  MockTimelineEventItem? timelineEventItem,
+}) {
+  return MockChatItem(
+    roomId: roomId,
+    typingUsers: typingUsers,
+    mockRoom: MockRoom(
+      mockRoomId: roomId,
+      mockDisplayName: displayName,
+      mockNotificationMode: notificationMode ?? 'all',
+      mockActiveMembersIds: activeMembersIds ?? [],
+    ),
+    mockConvo: MockConvo(
+      mockConvoId: roomId,
+      mockIsDm: isDm ?? false,
+      mockIsBookmarked: isBookmarked ?? false,
+      mockNumUnreadNotificationCount: unreadNotificationCount ?? 0,
+      mockNumUnreadMentions: unreadMentions ?? 0,
+      mockNumUnreadMessages: unreadMessages ?? 0,
+      mockTimelineItem: MockTimelineItem(
+        mockTimelineEventItem: timelineEventItem,
+      ),
+    ),
+  );
+}
+
+final mockChatItem1 = createMockChatItem(
```

Comment:
I wonder if we could give them a bit more descriptive names for the specific usecases they cover to make it easier to debug, update and debug them later.

like this be
```suggestion
final emilyDmMuted = createMockChatItem(
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2783/commits/93c8d6d960aaeb238bb18c9517ebbabd0bb5259b